Sources/Client/WebSocket/Provider/StarscreamWebSocketProvider.swift

```diff
@@ -20,7 +20,7 @@ final class StarscreamWebSocketProvider: WebSocketProvider {
         }
     }
     
-    var isConnected: Bool { webSocket.isConnected }
```

Comment:
Starting from `4.0.0` Starscream no longer has such API. Implemented it in the same way it is done in `URLSessionWebSocketProvider`